templates/assets/env-ecs.yml

```diff
@@ -391,6 +410,7 @@ Resources:
             echo "Initializing consul agent: ${ConsulServerAutoScalingGroup}"
 
             instance_arn=$(curl -s http://localhost:51678/v1/metadata | jq -r '. | .ContainerInstanceArn' | awk -F/ '{print $NF}' )
+            sed -i "s/INSTANCE_ARN/$instance_arn/g" /consul-check.sh
```

Comment:
Rather than `sed` the consul-check script, i'd prefer to have the script be able to determine his own instance ARN.  I understand this means copy/paste line 412 into the `consul-check.sh` script, but that would avoid the coupling between the two and allow the consul-check script to be more independent.

Reply:
Okay. Sounds plausible. I'll get the changes in